src/Data/Audience.elm

```diff
@@ -5917,3 +5924,46 @@ audiencesJSON =
         ]
     }
     """
+
+
+
+-- Decoders. Fairly trivial, except for the `audienceTypeDecoder`.
+
+
+audiencesDecoder : Decoder (List Audience)
+audiencesDecoder =
+    JD.field "data" (JD.list audienceDecoder)
+
+
+audienceDecoder : Decoder Audience
+audienceDecoder =
+    JD.map4 Audience
+        (JD.field "id" JD.int)
+        (JD.field "name" JD.string)
+        (JD.field "type" audienceTypeDecoder)
+        (JD.field "folder" (JD.maybe JD.int))
+
+
+{-| I went with looking at the "type" field, as that seemed to have all the
+needed info all in itself. I might be wrong with my reading of the backend
+developers' minds :) But it seems to be the "current" intended way to consume
+the API.
```

Comment:
If you're wrong then we have a bug in production:D Anyway I've derived the same thing and think it's the best solution as it prevents states of having `user` and `curated` audience at the same time.